test/linalg/cholesky.jl

```diff
@@ -66,10 +66,24 @@ for eltya in (Float32, Float64, Complex64, Complex128, BigFloat, Int)
         capds = cholfact(apds)
         @test inv(capds)*apds ≈ eye(n)
         @test abs((det(capds) - det(apd))/det(capds)) <= ε*κ*n
+        if eltya <: BlasReal
+            capds = cholfact!(copy(apds))
```

Comment:
We've had bugs in one-line children before. If the `cholesky` tests were taking up most of the test time I'd worry about it, but as is I don't really see the issue with spending an additional 0.01% of our test time on these.